### PR TITLE
adding serialVersionUID to serializable classes

### PR DESCRIPTION
**Pull request by h-haddad**

### CODE REVIEW

File: galite-core/src/main/kotlin/org/kopi/galite/base/ExtendedMessageFormat.kt [hunk 1]
```diff
@@ -418,5 +418,7 @@ class ExtendedMessageFormat : MessageFormat {
     // default behavior can't be hacked.
     /*package*/
     val NULL_REPRESENTATION = Any()
+
+    private const  val serialVersionUID = 0L
```
Comment (SNsir) [inline galite-core/src/main/kotlin/org/kopi/galite/base/ExtendedMessageFormat.kt:422]: delete space

File: galite-core/src/main/kotlin/org/kopi/galite/form/VFieldException.kt [hunk 2]
```diff
@@ -40,4 +40,8 @@ class VFieldException(val field: VField,
       field.setObject(field.block!!.activeRecord, newValue)
     }
   }
+
+  companion object {
+    private const  val serialVersionUID = 0L
```
Comment (SNsir) [inline galite-core/src/main/kotlin/org/kopi/galite/form/VFieldException.kt:45]: delete space before val

File: galite-core/src/main/kotlin/org/kopi/galite/form/VQueryNoRowException.kt [hunk 3]
```diff
@@ -25,4 +25,8 @@ import org.kopi.galite.visual.VException
  *
  * @param message the associated message. It's optional.
  */
-class VQueryNoRowException(message: String? = null) : VException(message)
+class VQueryNoRowException(message: String? = null) : VException(message) {
+  companion object {
+    private const  val serialVersionUID = 0L
```
Comment (SNsir) [inline galite-core/src/main/kotlin/org/kopi/galite/form/VQueryNoRowException.kt:30]: delete space before val

File: galite-core/src/main/kotlin/org/kopi/galite/form/VQueryOverflowException.kt [hunk 4]
```diff
@@ -25,4 +25,8 @@ import org.kopi.galite.visual.VException
  *
  * @param message the associated message. It's optional.
  */
-class VQueryOverflowException(message: String? = null) : VException(message)
+class VQueryOverflowException(message: String? = null) : VException(message) {
+  companion object {
+    private const  val serialVersionUID = 0L
```
Comment (SNsir) [inline galite-core/src/main/kotlin/org/kopi/galite/form/VQueryOverflowException.kt:30]: delete space before val

File: galite-core/src/main/kotlin/org/kopi/galite/form/VSkipRecordException.kt [hunk 5]
```diff
@@ -25,4 +25,8 @@ import org.kopi.galite.visual.VException
  *
  * @param message the associated message. It's optional.
  */
-class VSkipRecordException(message: String? = null as String?) : VException(message)
+class VSkipRecordException(message: String? = null) : VException(message) {
+  companion object {
+    private const  val serialVersionUID = 0L
```
Comment (SNsir) [inline galite-core/src/main/kotlin/org/kopi/galite/form/VSkipRecordException.kt:30]: delete space before val

File: galite-core/src/main/kotlin/org/kopi/galite/print/PSPrintException.kt [hunk 6]
```diff
@@ -37,4 +37,8 @@ class PSPrintException : PrintException {
    * @param     cause           the cause  (null value permited
    */
   constructor(message: String?, cause: Throwable?) : super(message, cause, EXC_FATAL)
+
+  companion object {
+    private const  val serialVersionUID = 0L
```
Comment (SNsir) [inline galite-core/src/main/kotlin/org/kopi/galite/print/PSPrintException.kt:42]: delete space before val

File: galite-core/src/main/kotlin/org/kopi/galite/report/MReport.kt [hunk 7]
```diff
@@ -968,6 +968,10 @@ class MReport : Constants, Serializable {
     }
   }
 
+  companion object {
+    private const  val serialVersionUID = 0L
```
Comment (SNsir) [inline galite-core/src/main/kotlin/org/kopi/galite/report/MReport.kt:972]: delete space before val

File: galite-core/src/main/kotlin/org/kopi/galite/report/VBaseRow.kt [hunk 8]
```diff
@@ -23,4 +23,8 @@ class VBaseRow(data: Array<Any?>) : VReportRow(data) {
    * Return the level of the node. For base rows it is always 0
    */
   override fun getLevel(): Int = 0
+
+  companion object {
+    private const  val serialVersionUID = 0L
```
Comment (SNsir) [inline galite-core/src/main/kotlin/org/kopi/galite/report/VBaseRow.kt:28]: delete space before val

File: galite-core/src/main/kotlin/org/kopi/galite/report/VGroupRow.kt [hunk 9]
```diff
@@ -110,4 +110,8 @@ class VGroupRow(data: Array<Any?>, private val level: Int) : VReportRow(data) {
       }
     }
   }
+
+  companion object {
+    private const  val serialVersionUID = 0L
```
Comment (SNsir) [inline galite-core/src/main/kotlin/org/kopi/galite/report/VGroupRow.kt:115]: delete space before val

File: galite-core/src/main/kotlin/org/kopi/galite/report/VNoRowException.kt [hunk 10]
```diff
@@ -25,4 +25,8 @@ import org.kopi.galite.visual.VRuntimeException
  *
  * @param message the associated message. It's optional
  */
-class VNoRowException(message: String? = null) : VRuntimeException(message)
+class VNoRowException(message: String? = null) : VRuntimeException(message) {
+  companion object {
+    private const  val serialVersionUID = 0L
```
Comment (SNsir) [inline galite-core/src/main/kotlin/org/kopi/galite/report/VNoRowException.kt:30]: delete space before val

File: galite-core/src/main/kotlin/org/kopi/galite/util/FaxException.kt [hunk 11]
```diff
@@ -26,20 +26,24 @@ open class FaxException : Exception {
...
   /**
    * Constructs a new exception with the specified detail message and cause.
    *
    * @param     cause           the cause  (null value permited
    */
-  constructor(cause: Throwable) : super(cause) {}
+  constructor(cause: Throwable) : super(cause)
+
+  companion object {
+    private const  val serialVersionUID = 0L
```
Comment (SNsir) [inline galite-core/src/main/kotlin/org/kopi/galite/util/FaxException.kt:47]: delete space before val

File: galite-core/src/main/kotlin/org/kopi/galite/util/PrintException.kt [hunk 12]
```diff
@@ -73,5 +73,7 @@ open class PrintException : Exception {
     const val EXC_UNKNOWN = 0
     const val EXC_FATAL = 1
     const val EXC_RETRY = 2
+
+    private const  val serialVersionUID = 0L
```
Comment (SNsir) [inline galite-core/src/main/kotlin/org/kopi/galite/util/PrintException.kt:77]: delete space before val

File: galite-core/src/main/kotlin/org/kopi/galite/visual/PropertyException.kt [hunk 13]
```diff
@@ -47,4 +47,8 @@ class PropertyException : Exception {
    * @param     cause           the cause  (null value permited
    */
   constructor(message: String, cause: Throwable) : super(message, cause)
+
+  companion object {
+    private const  val serialVersionUID = 0L
```
Comment (SNsir) [inline galite-core/src/main/kotlin/org/kopi/galite/visual/PropertyException.kt:52]: delete space before val

File: galite-core/src/main/kotlin/org/kopi/galite/visual/VExecFailedException.kt [hunk 14]
```diff
@@ -48,4 +48,8 @@ open class VExecFailedException : VException {
    * Constructs an exception with no message.
    */
   constructor() : super()
+
+  companion object {
+    private const  val serialVersionUID = 0L
```
Comment (SNsir) [inline galite-core/src/main/kotlin/org/kopi/galite/visual/VExecFailedException.kt:53]: delete space before val

File: galite-core/src/main/kotlin/org/kopi/galite/visual/VInterruptException.kt [hunk 15]
```diff
@@ -21,4 +21,8 @@ package org.kopi.galite.visual
 /**
  * Constructs an exception with a message.
  */
-class VInterruptException : Exception()
+class VInterruptException : Exception() {
+  companion object {
+    private const  val serialVersionUID = 0L
```
Comment (SNsir) [inline galite-core/src/main/kotlin/org/kopi/galite/visual/VInterruptException.kt:26]: delete space before val

File: galite-core/src/main/kotlin/org/kopi/galite/visual/VRuntimeException.kt [hunk 16]
```diff
@@ -50,4 +50,8 @@ open class VRuntimeException : RuntimeException {
    * Constructs an exception with no message.
    */
   constructor() : super()
+
+  companion object {
+    private const  val serialVersionUID = 0L
```
Comment (SNsir) [inline galite-core/src/main/kotlin/org/kopi/galite/visual/VRuntimeException.kt:55]: delete space before val

File: galite-core/src/main/kotlin/org/kopi/galite/visual/VRuntimeException.kt [hunk 17]
```diff
@@ -50,4 +50,8 @@ open class VRuntimeException : RuntimeException {
...
+  }
```
Comment (SNsir) [inline galite-core/src/main/kotlin/org/kopi/galite/visual/VRuntimeException.kt:56]: add 
companion object {
    private const val serialVersionUID = 0L
}
in InconsistencyException class

Comment (SNsir) [inline galite-core/src/main/kotlin/org/kopi/galite/visual/VRuntimeException.kt:56]: add
companion object {
private const val serialVersionUID = 0L
}
in VReportRow class

File: galite-core/src/main/kotlin/org/kopi/galite/visual/VExecFailedException.kt (hunk 14 above)
Comment (SNsir) [inline galite-core/src/main/kotlin/org/kopi/galite/visual/VExecFailedException.kt:53]: add
companion object {
private const val serialVersionUID = 0L
}
in VException class

**Review by SNsir — CHANGES_REQUESTED** [review]

**Review by SNsir — APPROVED** [review]

File: galite-core/src/main/kotlin/org/kopi/galite/list/ObjectFormatter.kt [hunk 18]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (hfazai) [inline galite-core/src/main/kotlin/org/kopi/galite/list/ObjectFormatter.kt:1]: remove this

File: galite-core/src/main/kotlin/org/kopi/galite/report/ColumnStyle.kt [hunk 19]
```diff
@@ -20,10 +20,11 @@ package org.kopi.galite.report
 
 import java.awt.Color
 import java.awt.Font
+import java.io.Serializable
 
 import org.kopi.galite.util.base.InconsistencyException
 
-class ColumnStyle {
+class ColumnStyle : Constants, Serializable {
```
Comment (hfazai) [inline galite-core/src/main/kotlin/org/kopi/galite/report/ColumnStyle.kt:27]: remove Constants

**Pull request merged**